README.md

```diff
@@ -36,71 +43,27 @@ relate to different resources in a Kubernetes-native way. Starboard provides
-or from pre-built binary releases.
-
-### From the Binary Releases
-
-Every [release][release] of Starboard provides binary releases for a variety of operating systems. These
-binary versions can be manually downloaded and installed.
-
-1. Download your [desired version][release]
-2. Unpack it (`tar -zxvf starboard_darwin_x86_64.tar.gz`)
-3. Find the `starboard` binary in the unpacked directory, and move it to its desired destination
-   (`mv starboard_darwin_x86_64/starboard /usr/local/bin/starboard`)
-
-From there, you should be able to run Starboard CLI commands: `starboard help`
-
-#### kubectl plugin
+Starboard can be run in two different modes:
 
-The Starboard CLI is compatible with [kubectl][kubectl] and is intended as [kubectl plugin][kubectl-plugins],
-but it's perfectly fine to run it as a stand-alone executable. If you rename the `starboard` executable to
-`kubectl-starboard` and if it's in your path, you can invoke it using `kubectl starboard`.
-
-You can also install Starboard as a kubectl plugin with the [Krew][krew] plugins manager:
-
-```
-$ kubectl krew install starboard
-$ kubectl starboard help
-```
-
-### From Source (Linux, macOS)
-
-Building from source is slightly more work, but is the best way to go if you want to test the latest (pre-release)
-version of Starboard.
-
-You must have a working Go environment.
-
-```
-$ git clone git@github.com:aquasecurity/starboard.git
-$ cd starboard
-$ make
-```
+* As a [command-line tool][starboard-cli], so you can trigger scans and view the risks in a `kubectl`-compatible way
+  or as part of your CI/CD pipeline.
+* As an [operator][starboard-operator] to automatically update security report resources in response to workload and
+  other changes on a Kubernetes cluster - for example, initiating a vulnerability scan when a new pod is started.
 
-If required, it will fetch the dependencies and cache them. It will then compile `starboard` and place it in
-`bin/starboard`.
-
-### Docker
-
-We also release the Docker image `docker.io/aqusec/starbaord` to run Starboard as a Docker container or to manually
-schedule Kubernetes scan Jobs in your cluster.
+You can read more about the motivations and use cases [here][aqua-starboard-blog] and join our discussions [here][discussions]. 
 
-```
-$ docker container run --rm docker.io/aquasec/starboard:0.4.0 version
-Starboard Version: {Version:0.4.0 Commit:dd8e49701c1817ea174061c8731fe5bdbfb73d93 Date:2020-09-21T09:36:59Z}
-```
+![](docs/images/starboard-cli-with-octant-demo.gif)
 
 ## Getting Started
 
 The easiest way to get started with Starboard is to use [Starboard CLI][starboard-cli], which allows scanning Kubernetes
 workloads deployed in your cluster.
 
+> **NOTE:** Even though manual scanning through the command-line is useful, it doesn't scale well with a huge number
+> of Kubernetes workloads and / or multi-tenant clusters. Therefore, the [Starboard Operator][starboard-operator]
```

Comment:
```suggestion
> **NOTE:** Even though manual scanning through the command-line is useful, the fact that it's not automated makes it less suitable with a large number
> of Kubernetes workloads. Therefore, the [Starboard Operator][starboard-operator]
```

I'm assuming that automation is the main motivator here - my suggestion is to avoid the possible interpretation that "this is written badly so even installing it it might cause problems in your large production cluster", which I don't think is the case